tests/operators/test_redshift_to_s3_operator.py

```diff
@@ -31,7 +32,11 @@ class TestRedshiftToS3Transfer(unittest.TestCase):
 
     @mock.patch("boto3.session.Session")
     @mock.patch("airflow.hooks.postgres_hook.PostgresHook.run")
-    def test_execute(self, mock_run, mock_session):
+    @parameterized.expand([
```

Comment:
You have applied this to the wrong place, check my suggested diff https://github.com/apache/airflow/pull/6396#issuecomment-557171010

Reply:
It should be the following:

```
	@parameterized.expand([
	       [True, "key/table_"],
	        [False, "key"],
	    ])
	@mock.patch("boto3.session.Session")
	@mock.patch("airflow.hooks.postgres_hook.PostgresHook.run")
	def test_execute(self, table_as_file_name, expected_s3_key, mock_run, mock_session,):
```

Reply:
Check the position of `@parameterized.expand([`